lib/api_account.go

```diff
@@ -4,204 +4,192 @@ import (
 	"fmt"
 	"net/http"
 
+	"github.com/gorilla/mux"
+
 	"boscoin.io/sebak/lib/block"
 	"boscoin.io/sebak/lib/common"
 	"boscoin.io/sebak/lib/error"
 	"boscoin.io/sebak/lib/observer"
-	"boscoin.io/sebak/lib/storage"
-	"github.com/gorilla/mux"
 )
 
 const GetAccountHandlerPattern = "/account/{address}"
 
-func GetAccountHandler(storage *sebakstorage.LevelDBBackend) http.HandlerFunc {
-
-	return func(w http.ResponseWriter, r *http.Request) {
-
-		vars := mux.Vars(r)
-		address := vars["address"]
-		if found, err := block.ExistBlockAccount(storage, address); err != nil {
-			http.Error(w, "Error reading request body", http.StatusInternalServerError)
-			return
-		} else if !found {
-			http.Error(w, http.StatusText(http.StatusNotFound), http.StatusNotFound)
-			return
-		}
-
-		var err error
-		var ba *block.BlockAccount
+func (api NetworkHandlerAPI) GetAccountHandler(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
```

Comment:
The handler function looks like changed a lot, however, actually, it is not changed a lot.
Could you make an intermediate commit to make it look good?

Reply:
like this
```
func (api NetworkHandlerAPI) GetAccountHandler(w http.ResponseWriter, r *http.Request) {
    {
        vars := mux.Vars(r)
        address := vars["address"]
        ...
    }
}
```

Reply:
or, just confirm that the logic is not changed.

Reply:
@kfangw Wow great, I did expect or think this kind of way. Thanks, @kfangw .